raiden-ts/src/raiden.ts

```diff
@@ -376,12 +376,9 @@ export class Raiden {
    *       <li>number index of a remote account loaded in provider
    *            (e.g. 0 for Metamask's loaded account)</li>
    *     </ul>
-   * @param storage - Storage/localStorage-like object from where to load and store current
```

Comment:
The parent object needs to stay, as per JSDoc, only the `storage.storage` member property is removed. You may need to adapt its description though.

Reply:
:heavy_check_mark: